src/tools/wix/BinderFileManager.cs

```diff
@@ -297,7 +300,7 @@ public virtual bool CompareFiles(string targetFile, string updatedFile)
 
                         for (int i = 0; i < targetReadLength; ++i)
                         {
-                            if (targetBuffer[i] != updatedBuffer[i])
+                            if (0 != memcmp(targetBuffer, updatedBuffer, targetReadLength))
```

Comment:
This is still inside the loop...? I'd like to see profiling data that it's an improvement before taking in an msvcrt function via P/Invoke.